test/cpp/tensorexpr/test_conv.cpp

```diff
@@ -0,0 +1,86 @@
+#include <gtest/gtest.h>
+#include "torch/csrc/jit/tensorexpr/ir_simplifier.h"
+#include "torch/csrc/jit/tensorexpr/loopnest.h"
+#include "torch/csrc/jit/tensorexpr/tensor.h"
+#include "torch/torch.h"
+
+namespace torch {
+namespace jit {
+
+namespace te = torch::jit::tensorexpr;
+namespace F = torch::nn::functional;
+
+void testConv2D() {
+  te::KernelScope kernel_scope;
+
+  // Input dimensions.
+  constexpr int N = 1;
+  constexpr int C = 3;
+  constexpr int H = 11;
+  constexpr int W = 11;
+
+  // Filter dimensions.
+  constexpr int K = 8;
+  constexpr int R = 3;
+  constexpr int S = 3;
+
+  // Output dims.
+  constexpr int OH = H - R + 1;
```

Comment:
I just saw your response. This is perfectly fine. But I think the ones with padding to keep the output size the same is also very popular. Might be worth to have another one reflecting that. 

The reason a mask is interesting for performance in that when the axis is reordered, the masks should be adjusted as well.

Reply:
Oh I totally think it’s worth having some padded (and strided, grouped, etc) convolution tests too.  I agree :-)